bot.py

```diff
@@ -1515,10 +1516,13 @@ async def handle_zoom_event(data: dict):
         state: ZoomMeetingState = app["zoom_meeting_messages"][meeting_id]
     except KeyError:
         return
+    if event == "meeting.ended":
```

Comment:
Moving this up ensures that subsequent "participant removed" events get ignored.